core/src/main/java/org/ehcache/EhcacheManager.java

```diff
@@ -168,8 +176,10 @@ void closeEhcache(final String alias, final Ehcache<?, ?> ehcache) {
     } else {
       caches.remove(alias);
       value.setCache(null);
+      LOGGER.error("Cache '" + alias + "' creation in EhcacheManager failed.");
```

Comment:
I am in favour of wrapping the failure into another exception with the message and throwing that instead of logging a message and throwing an exception that may not carry the cache alias in its message.